.github/workflows/run_quic_interop.yml

```diff
@@ -22,12 +22,30 @@ jobs:
         with:
          repository: 'quic-interop/quic-interop-runner'
          fetch-depth: 0
-      - name: Install dependencies
-        run: |
-          pip install -r requirements.txt
-          sudo add-apt-repository ppa:wireshark-dev/stable
-          sudo apt-get update
-          sudo apt-get install -y tshark
+      - name: Install python requirements
+        uses: Wandalen/wretry.action@master 
```

Comment:
I insist not to bring new GitHub actions if it's really needed.

Reply:
What about just pinning an exact commit hash? That should be safe.

Reply:
This action can be easily replaced with a simple shell script/

Reply:
```suggestion
        run: |
          for i in {1..3}; do pip install -r requirements.txt; sleep 5; done
```

Reply:
It needs a bit more than that, as you need to break the loop if the command succeeds, but yeah, we can do that

Reply:
```sh
for i in {1..3}; do pip install -r requirements.txt && break; sleep 5; done
```

Reply:
One of the possible examples:

```yaml
name: Install python requirements
run: |
  command() { pip install -r requirements.txt ; }
  for i in {1..3}; do command && break; sleep 5; done
```

Reply:
do you happen to recall if the default shell configuration includes set -e?  I.e. if any command within a shell fails, does the step immediately terminate with failure?

Reply:
It includes `-e`:
https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsshell

Reply:
ok, I did some testing in my local repo, and we seem to be safe re: bash'es set -e setting.  I've updated the PR